src/appkit.cs

```diff
@@ -6495,6 +6506,11 @@ partial interface NSFont : NSSecureCoding, NSCopying {
 		[Export ("monospacedSystemFontOfSize:weight:")]
 		[Internal]
 		IntPtr _MonospacedSystemFont (nfloat fontSize, nfloat weight);
+		
+		[Mac (10,16)]
+		[Static]
+		[Export ("preferredFontForTextStyle:options:")]
+		NSFont PreferredFontForTextStyle (string style, NSDictionary options);
```

Comment:
Unless this is an overload of an existing method I would use

```suggestion
		NSFont GetPreferredFont (string textStyle, NSDictionary options);
```